.github/workflows/tests.yaml

```diff
@@ -0,0 +1,26 @@
+name: Python tests
+
+on: [pull_request]
+
+jobs:
+    run-tests:
+        name: Run Python tests
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Check out Git repository
+              uses: actions/checkout@v2
+
+            - name: Set up Python
+              uses: actions/setup-python@v1
+              with:
+                python-version: 3.7
+
+            - name: Setup conda environment
+              run: |
+                $CONDA/bin/conda env update --file environment.yml --name base
+                $CONDA/bin/pip install -e .
+
+            - name: Run tests
+              run: |
+                $CONDA/bin/pytest tests/
```

Comment:
This works (and I think is what I use). If you have a need for more complex uses of Conda (or dependency caching) in Github Actions in the future I recommend https://github.com/marketplace/actions/setup-miniconda